### PR TITLE
Login submit

### PR DESCRIPTION
**Pull request by alejoaquili**
## Summary
Add validations and submit with navigation to smart component LoginContainer.

#### Nexus S
nxS

#### Nexus 4
nx4

#### Nexus 5X
nx5x

#### Nexus 6
nx6

## Trello Card
https://trello.com/c/lYnMgL84/35-n-93-funcionalidad-textinputs-y-login-submit

### CODE REVIEW

File: src/app/screens/Login/index.js [hunk 1]
```diff
@@ -1,14 +1,50 @@
...
+
+    onChangeEmail = email => this.setState({email:email});
+
+    onChangePassword = password => this.setState({password:password});
+
+    validateEmail = () => {
+        if(emailRegex.test(this.state.email)) {
+            this.setState({invalidEmail: false});
+        } else {
+            this.setState({invalidEmail: true});
```
Comment (guidoprinc) [inline src/app/screens/Login/index.js]: first destruct `email` from state and then:
`this.setState({ invalidEmail: !emailRegex.text(email) });`

File: src/app/screens/Login/index.js [hunk 2]
```diff
@@ -1,14 +1,50 @@
...
+        }
+    };
+
+    validatePassword = () => {
+        if(this.state.password.length >= 8) {
+            this.setState({invalidPassword: false});
+        } else {
+            this.setState({invalidPassword: true});
```
Comment (guidoprinc) [inline src/app/screens/Login/index.js]: same

File: src/app/screens/Login/styles.js [hunk 3]
```diff
@@ -33,23 +43,28 @@ export default  styles = StyleSheet.create({
...
+    enabledButtonText: {
         ...text
+    },
+    disabledButtonText: {
+        ...text,
+        color: gray
+    },
+    errorMessage: {
+        fontSize: SMALL,
+        color: red
```
Comment (guidoprinc) [inline src/app/screens/Login/styles.js]: smaller would be great

**Comment by sfernandez11** [comment]
@alejoaquili why did you merge this `PR`? I had some corrections to do about it.

**Pull request merged**